### DISCUSSION TITLE
What does the "Cookie Affected" column mean?

### DISCUSSION THREAD

**Discussion by maccoddington** [GoogleChromeLabs/ps-analysis-tool#569] [Support Forum] [answered]
When downloading a cookie report a column titled "Cookie Affected" appears, most of the cookies say "no". Does this mean the cookie is not affected by third party cookie deprecation? Should I still create a plan for these cookies?

**Comment by amedina** [comment, marked as answer]
When downloading an analysis report from PSAT, there is a "Cookie Affected" column which indicates cases of cookies having issues (i.e. blocked cookies). If it says "no" for a cookie, it means the cookie did not experience any issue during the loading of the page analyzed. 

If all the cookies indicate "no" in that column, it is very likely the analysis was done without [enabling the Chrome DevTools Protocol (CDP)](https://github.com/GoogleChromeLabs/ps-analysis-tool/wiki/PSAT-Settings-and-Permissions#enabling-chrome-devtools-protocol-in-psat) in PSAT. When CDP is not enabled, PSAT does not detect cookie blocking and therefore the "Cookie Affected" column will indicate "no" for all cookies.

In principle, the analysis flow should be to first look at PSAT's cookie insights Insights page where, if CDP hasn't been enabled, will show 0 cookies blocked and the message indicating that CDP must be enabled to see cookie blockings. Having said that, we could improve the download experience by reminding users if CDP is not enabled when they click on the download button. This will be included in our roadmap for an upcoming version.

**Reply by maccoddington** [reply]
Thank you!

Is the status for blocked cookies dependent on current settings for the browser? For example, if I do not currently have third party cookies blocked, cookies may return 'not affected', though those cookies may become blocked post third party cookie deprecation?

**Reply by milindmore22** [reply]
@maccoddington The PSAT extension helps you identify cookies that are affected by any issues. The issues can be caused by various reasons such as third-party cookie depreciation, users blocking the cookies etc. In the extension, we highlight such cookies in yellow color with the issue mentioned in the cookie details panel. (cookies are blocked when there is an issue with them, so this section is titled blocked reason)

Whereas, when we export the data to a CSV file, there is no possibility to highlight any row. Therefore, the “Cookie affected” column was added. Its value being true/yes denotes that a cookie is affected by an issue.

Third-party cookies are only affected in a 3PCD enabled browser. So, If you are trying to find cookies which will be affected by 3PCD, it's crucial to block third-party cookies in your browser as described [here](https://support.google.com/chrome/answer/95647)
